README.md

```diff
@@ -80,6 +80,28 @@ ERB
 visit_template source, layout: "custom"
 ```
 
+View components are supported through the `#render_component` helper:
+
+```ruby
+button = ButtonComponent.new(type: :submit) do
+  "Save me!"
+end
+
+render_component button
+```
+
+View component-based layouts are supported as well:
+
+```ruby
+button = ButtonComponent.new(type: :submit) do
+  "Save me!"
+end
+
+render_component button, layout: PreviewLayout
+# or
+render_component button, layout: -> { PreviewLayout }
```

Comment:
Maybe it's silly to support lambdas, but my initial assumption that that's how it would be expected by `#render` is because that's how we pass em to `ActionController#Base.layout`, but instead you get a pretty deep down in the process exception. I figured, you know, give folks what they're used to, avoid some frowns.